tests/sentry/processing/realtime_metrics/test_redis.py

```diff
@@ -380,90 +392,104 @@ def test_projects_mixed_buckets(
 #
 
 
-def test_get_counts_for_project_unset(store: RedisRealtimeMetricsStore) -> None:
+def test_get_counts_for_project_unset(store: RedisRealtimeMetricsStore, mock_time) -> None:
+    mock_time.return_value = 113.0
     counts = store.get_counts_for_project(42)
-    assert list(counts) == []
+
+    assert list(counts) == [BucketedCount(timestamp=110, count=0)]
```

Comment:
thank you for fixing up the timestamps so they respect the bucket sizes in the tests, by the way!